lib/generators/templates/setup/bin_setup.rb

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+require "fileutils"
+require "open3"
+require "bundler"
+
+# path to your application root.
+APP_ROOT = File.expand_path("..", __dir__)
+
+def system!(*args)
+  system(*args, exception: true)
+end
+
+def using_node?
+  File.exist? "package.json"
+end
+
+FileUtils.chdir APP_ROOT do
+  if ENV["CI"].nil?
```

Comment:
Is the idea that those of us in #not-macos need to identify as CI?

Reply:
I might be misunderstanding, but this is to prevent this block of code from running during GitHub actions, since `CI` is a [default environment variable](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables).

Is the concern that some operating systems ship with a `CI` value set?

Reply:
Those of us on Linux don't use Homebrew.

Reply:
Ahh, thank you for clarifying. It sounds like we should check to see if Homebrew is also installed too.

Reply:
Let's keep this application specific for now.

97068cd